test/cadet_web/controllers/user_controller_test.exs

```diff
@@ -12,8 +12,23 @@ defmodule CadetWeb.UserControllerTest do
   end
 
   describe "GET /user" do
-    test "success", %{conn: conn} do
+    test "success, student", %{conn: conn} do
```

Comment:
My bad, I think I missed this in my code review of the PR. Could you help me refactor this file to use tags instead?

```elixir
@tags authenticate: :student
```
Similarly for the staff below.

Reply:
Done